internal/integrationtest/testdata/boards.local.txt

```diff
@@ -0,0 +1,26 @@
+nessuno.name=Arduino Nessuno
+nessuno.vid.0=0x2341
+nessuno.pid.0=0x0043
+nessuno.vid.1=0x2341
+nessuno.pid.1=0x0001
+nessuno.vid.2=0x2A03
+nessuno.pid.2=0x0043
+nessuno.vid.3=0x2341
+nessuno.pid.3=0x0243
+nessuno.upload.tool=avrdude
+nessuno.upload.protocol=arduino
+nessuno.upload.maximum_size=32256
+nessuno.upload.maximum_data_size=2048
+nessuno.upload.speed=115200
+nessuno.bootloader.tool=avrdude
+nessuno.bootloader.low_fuses=0xFF
+nessuno.bootloader.high_fuses=0xDE
+nessuno.bootloader.extended_fuses=0xFD
+nessuno.bootloader.unlock_bits=0x3F
+nessuno.bootloader.lock_bits=0x0F
+nessuno.bootloader.file=optiboot/optiboot_atmega328.hex
+nessuno.build.mcu=atmega328p
+nessuno.build.f_cpu=16000000L
+nessuno.build.board=AVR_NESSUNO
```

Comment:
this file should also be removed from https://github.com/arduino/arduino-cli/blob/master/test/testdata/boards.local.txt

Reply:
I did not remove it in this PR because other tests, like [test_compile_part_4](https://github.com/arduino/arduino-cli/blob/master/test/test_compile_part_4.py) and [test_core](https://github.com/arduino/arduino-cli/blob/master/test/test_core.py), are still using it.